Automattic/wp-calypso
# Marketplace: Add a confetti animation to the post-purchase page

#### Proposed Changes

* Adds a confetti component to the post-purchase thank you page with a simple fading-in animation

|Static component | Animated Gif|
|-------|------|
|<img width="1283" alt="CleanShot 2023-01-10 at 14 02 23@2x" src="https://user-images.githubusercontent.com/3519124/211558735-7f8a2d41-5525-45e2-bb91-45ea9004027b.png">|![CleanShot 2023-01-10 at 14 11 27](https://user-images.githubusercontent.com/3519124/211561121-c3b09b3e-f992-483f-951c-d2b8182363b5.gif)|


#### Testing Instructions



* On a business or eCommerce site, install a plugin using the Marketplace or visiting `/plugins/<site_id>` or alternatively visit the following URL `/marketplace/thank-you/woocommerce-bookings/<site_id>`
* Make sure the thank you page contains the `Confetti` component in the screenshot above and you can see a simple fading-in animation

#### Pre-merge Checklist





Related to #70892
